ada_2_fhir-r4/mp/9.2.0/payload/dev/mp-MedicationUse2.xsl

```diff
@@ -117,23 +117,14 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                     </identifier>
                 </xsl:for-each>
 
+                <!-- AWE: MP-737 - no distinction between active/intended/completed needed -->
```

Comment:
Hmm... dit is een wijziging in payload/dev. Waarom hier en niet payload/1.0?

En ik zie dat mp_latest_package nog naar dev verwijst. Had dat dan niet 1.0 moeten zijn eigenlijk?

Reply:
Ik heb idd gewoon doorgeklikt vanaf latest package.

Reply:
Verplaatst en dev verwijderd